packages/react-provider/src/hooks/use-actions.ts

```diff
@@ -1,15 +1,20 @@
+import { sendClickedRequest } from '~/lib/events';
 import useCourier from './use-courier';
 
 export default () => {
-  const { dispatch } = useCourier();
+  const { dispatch, clientKey } = useCourier();
```

Comment:
ill make a ticket to make the toast reducer this week too since this is ambiguous what handleOnClick is

Reply:
Yes, I was thinking about what actions can go here that are agnostic to toast and inbox